tool/azure/azure-release-template.yml

```diff
@@ -25,12 +25,13 @@ extends:
           export SERVICE_BASE_PATH=$(SERVICE_BASE_PATH)
           export APIGEE_ENVIRONMENT=$(APIGEE_ENVIRONMENT)
 
-          # Poll deploying API until our deployed version matches the release version
+          # Poll EPS API until our deployed version of EPS matches the release version
```

Comment:
the logic behind this check should probably only be for pr builds (+dev) until releases are CI/CD